packages/web-forms/src/components/OdkWebForm.vue

```diff
@@ -98,9 +104,9 @@ watchEffect(() => {
 		display: flex;
 		flex-direction: column;
 		max-width: var(--wf-max-form-width);
+		min-height: calc(100vh - 5rem);
```

Comment:
As anticipated, the positioning is off in mobile Safari because of its dynamic viewport height. To address this...

1. These can go wherever we want global styles/variables, or can be defined on `.odk-form` instead:

```scss
:global(:root) {
	--viewport-height: 100vh;
}

@supports (height: 100dvh) {
	:global(:root) {
		--viewport-height: 100dvh;
	}
}

```

2. And then here:

```suggestion
		min-height: calc(var(--viewport-height) - 5rem);
```

<details>
<summary>Alternatively, with the fallback style directly inline</summary>

This doesn't require the `@supports` query.


```suggestion
		min-height: calc(100vh - 5rem);
		min-height: calc(100dvh - 5rem);
```
</details>

I will say that this got me wondering whether we want to use viewport height at all (it won't make sense if/when Web Forms is embedded in a host application's UI). But we can cross that bridge when we come to it.